pkg/aggregator/ckey/key_test.go

```diff
@@ -9,7 +9,7 @@ import (
 	"fmt"
 	"testing"
 
-	"github.com/DataDog/datadog-agent/pkg/tagset"
+	tagset "github.com/DataDog/datadog-agent/pkg/tagset/old"
```

Comment:
We use `tagset` in some places and `oldtagset` in others, what's the criteria?

Reply:
No real criteria, although I'll make it consistent.  I suspect goimport did this?  Anyway, all of this `old` stuff is going away in the next PR.